src/main.js

```diff
@@ -383,13 +386,14 @@ function handleAppWebContentsCreated(dc, contents) {
     const contentID = event.sender.id;
     const parsedURL = parseURL(url);
 
-    if (isTrustedURL(parsedURL)) {
+    if (isTrustedURL(parsedURL) || isTrustedPopupWindow(event.sender)) {
```

Comment:
This line allows navigation to outside domains from within the popup window. This is what was preventing the Google authentication button from working previously.